.github/workflows/ci-for-diff-ns.yaml

```diff
@@ -25,48 +23,73 @@ jobs:
-          kubectl rook_ceph -o test-operator -n test-cluster operator set ROOK_LOG_LEVEL DEBUG
-          kubectl rook_ceph -o test-operator -n test-cluster --context=$(kubectl config current-context) mons
-          kubectl rook_ceph -o test-operator -n test-cluster rook version
-          kubectl rook_ceph -o test-operator -n test-cluster rook status
-          kubectl rook_ceph -o test-operator -n test-cluster rook status all
-          kubectl rook_ceph -o test-operator -n test-cluster rook status cephobjectstores
+          kubectl rook-ceph -o test-operator -n test-cluster operator set ROOK_LOG_LEVEL DEBUG
+          kubectl rook-ceph -o test-operator -n test-cluster --context=$(kubectl config current-context) mons
+          kubectl rook-ceph -o test-operator -n test-cluster rook version
+          kubectl rook-ceph -o test-operator -n test-cluster rook status
+          kubectl rook-ceph -o test-operator -n test-cluster rook status all
+          kubectl rook-ceph -o test-operator -n test-cluster rook status cephobjectstores
           # to allow time for reconcile, sleep before listing the pools
           sleep 5
-          kubectl rook_ceph -o test-operator -n test-cluster rbd ls replicapool
+          kubectl rook-ceph -o test-operator -n test-cluster rbd ls replicapool
 
           # for testing osd purge scale the osd deplyment
           kubectl --namespace test-cluster scale deploy/rook-ceph-osd-0 --replicas=0
           # we need to sleep so the osd will be marked down before purging the osd
           sleep 5
-          kubectl-rook_ceph -o test-operator -n test-cluster rook purge-osd 0 --force
-          kubectl rook_ceph -o test-operator -n test-cluster health
+          kubectl rook-ceph -o test-operator -n test-cluster rook purge-osd 0 --force
+          kubectl rook-ceph -o test-operator -n test-cluster health
 
           # for testing start-debug and stop-debug deployment
           # mon
-          kubectl rook_ceph -o test-operator -n test-cluster debug start rook-ceph-mon-a
+          kubectl rook-ceph -o test-operator -n test-cluster debug start rook-ceph-mon-a
           # sleep till the debug pod get running
           sleep 5
-          kubectl rook_ceph -o test-operator -n test-cluster debug stop rook-ceph-mon-a
+          kubectl rook-ceph -o test-operator -n test-cluster debug stop rook-ceph-mon-a
           # osd
-          kubectl rook_ceph -o test-operator -n test-cluster debug start rook-ceph-osd-0
+          kubectl rook-ceph -o test-operator -n test-cluster debug start rook-ceph-osd-0
           # sleep till the debug pod get running
           sleep 5
-          kubectl rook_ceph -o test-operator -n test-cluster debug stop rook-ceph-osd-0
+          kubectl rook-ceph -o test-operator -n test-cluster debug stop rook-ceph-osd-0
+  # This test is required to test latest changes or the changes that not present
+  # with current version of rook-ceph krew plugin
+  with-pr-changes-in-custom-namespace:
```

Comment:
This is a difficult pattern to maintain if every new feature is added to this section, then it merges, then included in a release, then we need a separate PR to move it to the other section. What if we just leave everything in the CI in the non-official krew commands using `rook_ceph`, and then no need to update the CI later after new features are added? Or is there a benefit of converting the CI like this other than using the exact syntax as the docs?

Reply:
no, I don't think we have a benefit here. So, should we close this pr or any other changes you are suggesting?

Reply:
Yeah how about if we close this PR and leave the tests with their current pattern